runtime/src/main/java/com/xatkit/plugins/moodle/platform/io/MoodleIntentProvider.java

```diff
@@ -33,12 +33,11 @@ public MoodleIntentProvider(MoodlePlatform runtimePlatform, Configuration config
         super(runtimePlatform, configuration);
         this.runtimePlatform.getSocketIOServer().addEventListener(SocketEventTypes.USER_MESSAGE.label, MessageObject.class, 
                 (socketIOClient, messageObject, ackRequest)-> {
-                        Log.info("Received message {0}", messageObject.getMessage());
-                        Log.info("Received username {0}", messageObject.getUsername());
-                        String username = messageObject.getUsername();
-                        String channel = socketIOClient.getSessionId().toString();
+                        Log.info("Received message: {0}", messageObject.getMessage());
+                        Log.info("Received userId: {0}", messageObject.getUserId());
+                        String username = messageObject.getUserId();
```

Comment:
Variable should be named `userId` instead of `username`

Reply:
resolved in next commit